src/creatures/creatures_definitions.hpp

```diff
@@ -714,11 +714,11 @@ enum ChannelEvent_t : uint8_t {
 	CHANNELEVENT_EXCLUDE = 3,
 };
 
-enum VipStatus_t : uint8_t {
-	VIPSTATUS_OFFLINE = 0,
-	VIPSTATUS_ONLINE = 1,
-	VIPSTATUS_PENDING = 2,
-	VIPSTATUS_TRAINING = 3
+enum class VipStatus_t : uint8_t {
+	OFFLINE = 0,
```

Comment:
It's not mandatory, but I would say that following an Uppercase pattern for the first letter of each word looks better.